### PR TITLE
feat: send on enter

### PR DESCRIPTION
**Pull request by koriwi**
tested locally. works as expected

### CODE REVIEW

**Comment by pedrolamas** [comment]
Thank you @koriwi, can you please add a footer to the main PR comment with a "Signed-off-by" line, following the [CONTRIBUTING](https://github.com/fluidd-core/fluidd/blob/develop/CONTRIBUTING.md) guidelines.

Thank you!

**Comment by koriwi** [comment]
@pedrolamas should be good now

**Review by pedrolamas — CHANGES_REQUESTED** [review]
This indeed works, though I think an optimal (and consistent with the rest of the codebase) will be encapsulate the `<v-card>` element on with a `<v-form>`, add `@submit.prevent="$emit('click', runCommand)"` to it, and then remove the click handler from the button and set it `type="submit"`

**Comment by koriwi** [comment]
Will do soon. As I'm not a Vue dev (more or less react) and just changed the steppers on my delta ;)

**Comment by koriwi** [comment]
done

File: src/components/ui/AppMacroBtn.vue [hunk 1]
```diff
@@ -46,49 +46,51 @@
...
+                  </app-btn>
+                </template>
+              </v-text-field>
+            </v-layout>
+          </v-card-text>
+          <v-divider />
+          <v-card-actions class="px-3 py-3">
+            <app-btn
+              block
+              @click="$emit('click', runCommand)"
```
Comment (pedrolamas) [inline src/components/ui/AppMacroBtn.vue:87]:
```suggestion
              type="submit"
```

**Review by pedrolamas — APPROVED** [review]

**Review by matmen — APPROVED** [review]

**Pull request merged**